docs/configuration.rst

```diff
@@ -177,6 +177,14 @@ These settings have effect only in tooltips.
    Type: string
 
 
+.. confval:: hoverxref_tooltip_lazy
+   Description: Whether to lazily generate tooltips (insert the html on hover, rather than on page load). This is known to be slower, but prevents the browser from stalling on load for very big doc pages.
```

Comment:
```suggestion
   Description: Whether to lazily generate tooltips (insert the HTML for the tooltip on hover, rather than on page load).
   This is known to be slower in general, but prevents the browser from stalling on load for very big pages.
   We recommend you keeping it as `False` unless you are experimenting some page load issues.
```